appeals/web/src/server/appeals/appeal-details/appeal-details.mapper.js

```diff
@@ -401,7 +401,7 @@ function removeAccordionComponentsActions(accordionComponents) {
 				);
 				break;
 			default:
-				removeActions(component);
+				removeSummaryListActions(component);
```

Comment:
This should probably be removed to avoid issues, as it should only be used on summary-list components

Reply:
ok removed!